src/protean/core/entity.py

```diff
@@ -305,6 +313,48 @@ def all(self):
 
         return results
 
+    def update(self, *data, **kwargs):
```

Comment:
This is not how we should do this as there will be a performance issue. We should just pass the query to the _delete_objects and the _update_objects methods and let them deal with it.

Reply:
This is how Django does it, because there will be callback hooks in the future for Entity lifecycle events. We can provide two mechanisms for people to sidestep if this is an issue:

Reply:
Yes we would need it as the main drawbacks of such frameworks is performance and so we need to give a way for users to overcome it

Reply:
👍 #103 mentions the issue. Please go ahead and merge this PR.